src/sentry/api/endpoints/group_notes.py

```diff
@@ -97,5 +101,28 @@ def post(self, request, group):
             data=data,
         )
 
+        # sync Sentry comments to external issues
+        if features.has('organizations:internal-catchall', group.organization, actor=request.user):
+            external_issues = list(
+                ExternalIssue.objects.filter(
+                    id__in=GroupLink.objects.filter(
+                        project_id=group.project_id,
+                        group_id=group.id,
+                        linked_type=GroupLink.LinkedType.issue,
+                    ).values_list('linked_id', flat=True)
+                )
+            )
+
+            if external_issues:
+                integrations = {
+                    i.id: i for i in Integration.objects.filter(
+                        id__in=[external_issue.integration_id for external_issue in external_issues]
+                    )
+                }
+
+                for external_issue in external_issues:
```

Comment:
So this is fine because it's behind a switch, but this needs to be moved to a celery task. For future reference, all these kinds of syncing should be done async and not in process of the web request.

This is going to make clicking "Submit" take multiple seconds, or more, depending on the server we're talking to, and the number of them, since this is talking to all configured integrations.

Reply:
ok, then if it's cool with you, i'll just merge as is and follow up with moving to a task within the next day or two after i finish what i'm currently working on